core/internal/api/send.go

```diff
@@ -65,7 +65,15 @@ func (client *clientImpl) sendToWandbBackend(
 ) (*http.Response, error) {
 	client.setClientHeaders(req)
 	client.setAuthHeaders(req)
-	return client.send(req)
+
+	resp, err := client.send(req)
+
+	// This is a bug that happens with retryablehttp sometimes.
```

Comment:
would be good to eventually figure out why this happens.
do you think it is a retryablehttp issue?